### ISSUE TITLE
[EasybytezCom] Problems for registered account

### ISSUE THREAD

**Issue by Guidobelix** [pyload/pyload#914]
As a registered user I get the following problem while downloading from Easybytez.com:
22.11.2014 16:32:13 INFO      Download starts: http://www.easybytez.com/xxxxxxxxxxxx/XXX.Xxx.Xxxxx.xx.XXXXxx.XxxX.xxxxxx.xxx
22.11.2014 16:32:13 DEBUG     EasybytezCom: Load url: http://www.easybytez.com/xxxxxxxxxxxx | decode=True | cookies=True | save_cookies=True | get={} | follow_location=True | just_header=False | post={} | ref=True
22.11.2014 16:32:14 DEBUG     EasybytezCom: File info (before update): {}
22.11.2014 16:32:14 DEBUG     EasybytezCom: File info (after update): {'status': 2, 'S': '260.0', 'U': 'MB', 'name': 'CSI Las Vegas DVDRip XviD part06 rar', 'url': u'http://www.easybytez.com/xxxxxxxxxxxx', 'N': 'XXX Xxx Xxxxx XXXXxx XxxX xxxxxx xxx', 'size': 272629760.0}
22.11.2014 16:32:14 DEBUG     EasybytezCom: File name: XXX Xxx Xxxxx XXXXxx XxxX xxxxxx xxx | File size: 272629760.0
22.11.2014 16:32:14 DEBUG     EasybytezCom: Handled as free download
22.11.2014 16:32:14 DEBUG     EasybytezCom: Getting download link: #1
22.11.2014 16:32:14 DEBUG     EasybytezCom: {'method_premium': 'Premium Download', 'usr_login': '', 'referer': '', 'fname': 'CSI.Las.Vegas.1x.DVDRip.XviD.part06.rar', 'method_free': 'Free Download', 'id': 'zbabyfb3la0k', 'op': 'download1'}
22.11.2014 16:32:14 DEBUG     EasybytezCom: ReCaptcha key not found
22.11.2014 16:32:14 DEBUG     EasybytezCom: SolveMedia key not found
22.11.2014 16:32:14 INFO      EasybytezCom: Wait: -1416670334 seconds | Reconnect: False
22.11.2014 16:32:14 DEBUG     EasybytezCom: Ignore reconnection due account logged
22.11.2014 16:32:14 DEBUG     EasybytezCom: Load url: http://www.easybytez.com/xxxxxxxxxxxx | decode=True | cookies=True | save_cookies=True | get={} | follow_location=True | just_header=False | post={'usr_login': '', 'referer': '', 'fname': 'XXX.Xxx.Xxxxx.xx.XXXXxx.XxxX.xxxxxx.xxx', 'method_free': 'Free Download', 'id': 'zbabyfb3la0k', 'op': 'download1'} | ref=True
22.11.2014 16:32:14 DEBUG     EasybytezCom: Getting download link: #2
22.11.2014 16:32:14 DEBUG     EasybytezCom: {'rand': '57tn2gqfkhtavr4kvl3qb6zql7azzb6ugwyym3i', 'method_premium': '', 'down_script': '1', 'referer': '', 'method_free': 'Free Download', 'id': 'xxxxxxxxxxxx', 'op': 'download2'}
22.11.2014 16:32:14 DEBUG     EasybytezCom: Set waitUntil to: 1416670350.398965 (previous: 0.000000) | Wait: 16 seconds
22.11.2014 16:32:14 DEBUG     EasybytezCom: Set wantReconnect to: False (previous: False)
22.11.2014 16:32:14 DEBUG     EasybytezCom: ReCaptcha key not found
22.11.2014 16:32:14 DEBUG     EasybytezCom: SolveMedia key not found
22.11.2014 16:32:14 INFO      EasybytezCom: Wait: 15 seconds | Reconnect: False
22.11.2014 16:32:14 DEBUG     EasybytezCom: Ignore reconnection due account logged
22.11.2014 16:32:30 DEBUG     EasybytezCom: Load url: http://www.easybytez.com/xxxxxxxxxxxx | decode=True | cookies=True | save_cookies=True | get={} | follow_location=True | just_header=False | post={'rand': '57tn2gqfkhtavr4kvl3qb6zql7azzb6ugwyym3i', 'down_script': '1', 'referer': '', 'method_free': 'Free Download', 'id': 'xxxxxxxxxxxx', 'op': 'download2'} | ref=True
22.11.2014 16:32:30 WARNING   Download failed: XXX Xxx Xxxxx XXXXxx XxxX xxxxxx xxx | Parse error: Direct download link detected but not found | Plugin may be out of date
Traceback (most recent call last):
  File "/home/guidobelix/pyload-git/module/PluginThread.py", line 189, in run
    pyfile.plugin.preprocessing(self)
  File "/home/guidobelix/pyload-git/module/plugins/Plugin.py", line 260, in preprocessing
    return self.process(self.pyfile)
  File "/home/guidobelix/pyload-git/module/plugins/internal/SimpleHoster.py", line 336, in process
    self.handleFree()
  File "/home/guidobelix/pyload-git/module/plugins/internal/XFSHoster.py", line 88, in handleFree
    link = self.getDownloadLink()
  File "/home/guidobelix/pyload-git/module/plugins/internal/XFSHoster.py", line 133, in getDownloadLink
    self.error(_("Direct download link detected but not found"))
  File "/home/guidobelix/pyload-git/module/plugins/internal/SimpleHoster.py", line 505, in error
    return _error(self, reason, type)
  File "/home/guidobelix/pyload-git/module/plugins/internal/SimpleHoster.py", line 31, in _error
    raise Fail(msg)
Fail: Parse error: Direct download link detected but not found | Plugin may be out of date
22.11.2014 16:32:30 INFO      UnSkipOnFail: look for skipped duplicates for CSI Las Vegas DVDRip XviD part06 rar (pid:16)
22.11.2014 16:32:31 INFO      Download starts: http://www.easybytez.com/yyyyyyyyyyyy/YYY.Yyy.Yyyyy.yy.YYYYyy.YyyY.yyyyyy.yyy
22.11.2014 16:32:31 DEBUG     EasybytezCom: Load url: http://www.easybytez.com/yyyyyyyyyyyy | decode=True | cookies=True | save_cookies=True | get={} | follow_location=True | just_header=False | post={} | ref=True
22.11.2014 16:32:32 DEBUG     EasybytezCom: File info (before update): {}
22.11.2014 16:32:32 DEBUG     EasybytezCom: File info (after update): {'status': 2, 'S': '260.0', 'U': 'MB', 'name': 'CSI Las Vegas DVDRip XviD part04 rar', 'url': u'http://www.easybytez.com/yyyyyyyyyyyy', 'N': 'YYY Yyy Yyyyy YYYYyy YyyY yyyyyy yyy', 'size': 272629760.0}
22.11.2014 16:32:32 DEBUG     EasybytezCom: File name: YYY Yyy Yyyyy YYYYyy YyyY yyyyyy yyy | File size: 272629760.0
22.11.2014 16:32:32 DEBUG     EasybytezCom: Handled as free download
22.11.2014 16:32:32 DEBUG     EasybytezCom: Getting download link: #1
22.11.2014 16:32:32 DEBUG     EasybytezCom: {'method_premium': 'Premium Download', 'usr_login': '', 'referer': '', 'fname': 'YYY.Yyy.Yyyyy.yy.YYYYyy.YyyY.yyyyyy.yyy', 'method_free': 'Free Download', 'id': 'yyyyyyyyyyyy', 'op': 'download1'}
22.11.2014 16:32:32 DEBUG     EasybytezCom: ReCaptcha key not found
22.11.2014 16:32:32 DEBUG     EasybytezCom: SolveMedia key not found
22.11.2014 16:32:32 INFO      EasybytezCom: Wait: -1416670352 seconds | Reconnect: False
22.11.2014 16:32:32 DEBUG     EasybytezCom: Ignore reconnection due account logged
22.11.2014 16:32:32 DEBUG     EasybytezCom: Load url: http://www.easybytez.com/yyyyyyyyyyyy | decode=True | cookies=True | save_cookies=True | get={} | follow_location=True | just_header=False | post={'usr_login': '', 'referer': '', 'fname': 'YYY.Yyy.Yyyyy.yy.YYYYyy.YyyY.yyyyyy.yyy', 'method_free': 'Free Download', 'id': 'yyyyyyyyyyyy', 'op': 'download1'} | ref=True
22.11.2014 16:32:32 DEBUG     EasybytezCom: Getting download link: #2
22.11.2014 16:32:32 WARNING   EasybytezCom: You have to wait 51 minutes, 58 seconds till next download  Download files instantly with <a href=
22.11.2014 16:32:32 DEBUG     EasybytezCom: Set waitUntil to: 1416673471.461714 (previous: 0.000000) | Wait: 3119 seconds
22.11.2014 16:32:32 DEBUG     EasybytezCom: Set wantReconnect to: True (previous: False)
22.11.2014 16:32:32 INFO      EasybytezCom: Wait: 3118 seconds | Reconnect: True
22.11.2014 16:32:32 DEBUG     EasybytezCom: Ignore reconnection due account logged

So, basically pyload tries do download the file but it fails with an error.
The other weid thing is that when it starts the following download in the cue, Easybytez answers with a wait time, i.e. from Easybytez perspective the previous download started, but there is no sign of any file on my drive.
